src/auto-calibrated-device.h

```diff
@@ -28,6 +28,14 @@ namespace librealsense
             float target_w, float target_h, rs2_update_progress_callback_sptr progress_callback) = 0;
         virtual std::string get_calibration_config() const = 0;
         virtual void set_calibration_config(const std::string& calibration_config_json_str) const = 0;
+
+        void add_depth_write_observer( std::function< void() > callback ) { _depth_write_callbacks.push_back( callback ); }
```

Comment:
Don't we want to input the new table?
For future use?

Reply:
The new table is returned by the calibration functions (run_on_chip_calibration etc...)
The user may decide to write the table or discard. When writing want to notify internal users of the table of the change.